docs/cmdline-opts/etag-compare.d

```diff
@@ -0,0 +1,17 @@
+Long: etag-compare
+Arg: <file>
+Help: Pass an ETag from a file as a custom header
+Protocols: HTTP
+---
+This option can be used to read an ETag from a file, and make
+an HTTP request conditional by sending a custom If-None-Match header
+using the extracted ETag.
+
+This option expects the first line of a file to be the desired ETag,
+everything after that is not considered. Empty files are not allowed.
```

Comment:
As you mention empty files. What happens more exactly when you specify an empty file? Does it cause an error?